amnestywebsite/humanity-theme
# Add new Hero block to replace Header and Banner blocks

Ref: https://github.com/amnestywebsite/amnesty-wp-theme/issues/1070, https://github.com/amnestywebsite/amnesty-wp-theme/issues/1538
Recreated from: https://github.com/amnestywebsite/amnesty-wp-theme/pull/2783

**Steps to test**:
1.  Create a post, and add the Hero block
2. Add some content (header, button text/link, background image etc)
3. Save and view the front end
4. Check the Hero displays and settings and content match that of the editor
5. Return to the editor, alter the settings and add an inner donation block.
6. Save and view on front end again.

**Considerations**:
RTL styles have been added and should be tested for all alignments of the Hero block

**Example**
http://bigbite.im/v/MkUBWd
